query_optimizer/resolver/Resolver.cpp

```diff
@@ -209,6 +227,26 @@ struct Resolver::QueryAggregationInfo {
   std::vector<E::AliasPtr> aggregate_expressions;
 };
 
+struct Resolver::WindowPlan {
+  explicit WindowPlan(E::WindowInfo window_info_in,
```

Comment:
No need for `explicit`, and we may want to add a reference to `window_info_in`, or use a rvalue instead.